applications/luci-app-mwan3/luasrc/model/cbi/mwan/globalsconfig.lua

```diff
@@ -3,61 +3,42 @@
 
 local net = require "luci.model.network".init()
 
-local s, m, mask, rtmon, rtlookup, logging, loglevel
+local s, m, o
 
 m = Map("mwan3", translate("MWAN - Globals"))
 
 s = m:section(NamedSection, "globals", "globals", nil)
 
-mask = s:option(
-	Value,
-	"mmx_mask",
+o = s:option(Value, "mmx_mask",
 	translate("Firewall mask"),
 	translate("Enter value in hex, starting with <code>0x</code>"))
-mask.datatype = "hex(4)"
-mask.default = "0xff00"
+o.datatype = "hex(4)"
+o.default = "0xff00"
```

Comment:
While you are fixing default values:

etc/config/mwan3 has a default of 0x3F00, and the docs reflect that ("The firewall mask (default 0x3F00) which is used to mark the outgoing package can be configured in the /etc/config/mwan3 globals section"), so this should be changed as well. Though, the docs are a bit inconsistent here, since there are many references to 250 interfaces being supported, when in reality the number of supported devices is FWMASK - 4 for the reserved marks (0 for unmarked, FWMASK for default, FWMASK - 1 for unreachable and FWMASK - 2 for blackhole).

Reply:
You're right, of course. We should change the default here.

In the past this was fixed to 250 and could not be changed.

@jamesmacwhite if you have time could you also update the docs for mmx_mask as mentioned by @aaronjg?

Reply:
Default value update for **mmx_mask** done with commit https://github.com/openwrt/luci/pull/4349/commits/9c99b42f034e0b14363c28f993db59ac9809c845

Reply:
@feckert The globals configuration was entirely absent from the docs until recently, but that should be in place already here with the correct `mmx_mask`: https://openwrt.org/docs/guide-user/network/wan/multiwan/mwan3#globals_configuration.

Realistically, to avoid the wrong number, should the exact value just be dropped in the docs and simply mention mwan3 supports a large amount of both physical and logical network interfaces. Who's actually going to be able to configure 250+ interfaces in a single deployment?!

There is a reference to this statement in LuCI btw (252 interfaces):

![image](https://user-images.githubusercontent.com/8067792/92468693-74e45e80-f1cb-11ea-86f0-6da0f12cd7ee.png)

Reply:
Thanks max interface hint fixed with commit https://github.com/openwrt/luci/pull/4349/commits/a7036258f9d8d89ae03014c527a4406c8fcd80f4

Reply:
I think the 252 is correct for a mask of 0xFF00. You can have fwmarks from 0x0100 to 0xFC00. Though, I'm not sure that hint is correct. Conceivably if someone wanted more interfaces for whatever reason, they could change the mask to 0XFFFF and add even more interfaces.

Reply:
@aaronjg @feckert I think the interface count seems pretty ambiguous, maybe it shouldn't be promoted with an exact number. The amount of interfaces the average use case of mwan3, surely wouldn't be an issue?

Reply:
Yeah, I think that line could safely be removed. The top line ("There are currently X of Y supported interfaces configured") has the correct number of interfaces, so the "MWAN supports up to..." is not really needed.